qa/tasks/ceph.py

```diff
@@ -1264,7 +1264,11 @@ def osd_scrub_pgs(ctx, config):
         unclean = [stat['pgid'] for stat in stats if 'active+clean' not in stat['state']]
         split_merge = []
         osd_dump = manager.get_osd_dump_json()
-        split_merge = [i['pool_name'] for i in osd_dump['pools'] if i['pg_num'] != i['pg_num_target']]
+        try:
+            split_merge = [i['pool_name'] for i in osd_dump['pools'] if i['pg_num'] != i['pg_num_target']]
+        except KeyError:
+            # mimic does not support 'pg_num_target'
```

Comment:
perhaps we could use the same comment "we don't support pg_num_target before nautilus" here as well.

Reply:
updated and repushed.